bin/hammer-complete

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+
+require 'English'
+@project_root = File.expand_path('../..', __FILE__)
+$LOAD_PATH.unshift(File.join(@project_root, 'lib'))
+HAMMER = ENV['HAMMER'] || File.join(@project_root, 'bin/hammer')
+
+require 'yaml'
+
+completion_cache_file = ENV['HAMMER_COMPLETION_CACHE'] || '~/.cache/hammer_completion.yml'
```

Comment:
`cli_config.yml` is loaded into ENV and we pull the value for cache location from there or use default, right?